.github/workflows/docker_login.yml

```diff
@@ -0,0 +1,23 @@
+# ------------------------------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------------------------------
+
+name: Docker login
+
+on:
+  push:
+    branches:
+      - master
+      - tanvi/new_branch
```

Comment:
Please, remove your branch from the GH file.

Reply:
The idea was to keep this specifically for docker account monitoring. For ex, doing login, adding rate check etc. 

But now that I see this : https://github.com/dapr/quickstarts/blob/feature/new_quickstarts/.github/workflows/build.yml looks like build workflow does have docker login in it, maybe now it makes more sense to check the limit instead cause maybe we are running into pull limits